pkgs/development/python-modules/rdkit/default.nix

```diff
@@ -19,28 +19,29 @@
 }:
 let
   external = {
-    avalon = fetchzip {
-      url = "http://sourceforge.net/projects/avalontoolkit/files/AvalonToolkit_1.2/AvalonToolkit_1.2.0.source.tar";
-      sha256 = "0nhxfxckb5a5qs0g148f55yarhncqjgjzcvdskkv9rxi2nrs7160";
-      stripRoot = false;
+    avalon = fetchFromGitHub {
+      owner = "rohdebe1";
+      repo = "ava-formake";
+      rev = "AvalonToolkit_2.0.2";
+      hash = "sha256-YI39OknHiSyArNGqRKrSVzEJnFc1xJ0W3UcTZrTKeME=";
     };
     yaehmop = fetchFromGitHub {
       owner = "greglandrum";
       repo = "yaehmop";
-      rev = "cfb5aeebbdf5ae93c4f4eeb14c7a507dea54ae9e";
-      sha256 = "sha256-QMnc5RyHlY3giw9QmrkGntiA+Srs7OhCIKs9GGo5DfQ=";
+      rev = "v2022.09.1";
+      hash = "sha256-QMnc5RyHlY3giw9QmrkGntiA+Srs7OhCIKs9GGo5DfQ=";
     };
     freesasa = fetchFromGitHub {
       owner = "mittinatten";
       repo = "freesasa";
-      rev = "2.1.1";
-      sha256 = "sha256-fUJvLDTVhpBWl9MavZwp0kAO5Df1QuHEKqe20CXNfcg=";
+      rev = "2.0.3";
```

Comment:
Note: downgrade to version specified in https://github.com/rdkit/rdkit/blob/dc16d0e160033ac215b574beed12a52d0b344fc5/External/FreeSASA/CMakeLists.txt#L15